addons/website_slides/security/website_slides_security.xml

```diff
@@ -90,15 +117,62 @@
             <field name="domain_force">[(1, '=', 1)]</field>
         </record>
 
+        <record id="rule_slide_slide_logged_user" model="ir.rule">
```

Comment:
Could you push the rule below the "public" one, similarly to how it was ordered for channels just above?